pkg/report/guilty.go

```diff
@@ -52,3 +55,23 @@ nextFile:
 	}
 	return ""
 }
+
+func GetMaintainers(linux string, file string) ([]string, error) {
+	// get_maintainer.pl is a Linux kernel script.
+	cmd := exec.Command("./scripts/get_maintainer.pl", "-f", file)
+	cmd.Dir = linux
+	stderr := new(bytes.Buffer)
+	stdout := new(bytes.Buffer)
+	cmd.Stdout = stdout
+	cmd.Stderr = stderr
+	if err := cmd.Run(); len(stdout.Bytes()) == 0 {
+		return nil, fmt.Errorf("get_maintainer.pl failed: %v\n%v\n%v\n", err, stdout.String(), stderr.String())
+	}
+	parts := strings.Split(stdout.String(), "\n")
+	var mtrs []string
+	for _, part := range parts {
+		mtr := strings.Split(part, " (")[0]
```

Comment:
I would prefer if we don't collect real names. That's unnecessary personal information.